keystrokes/src/components/Home/Home.js

```diff
@@ -1,16 +1,53 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import Game from '../Game/Game';
 import Nav from '../NavBar/NavBar';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import { ref } from '../../config/constants';
+import CharacterList from '../Character/CharacterList';
+
+function Home(props) {
+	const userId = props.user.email.replace(/[.$#[\]]/g, '');
+	const userPath = `/Users/${userId}`;
+	const [user, setUser] = useState({characters: {}, currentCharacter: null});
+
+	useEffect(() => {
+		const collectUserInfo = async () => {
+			const snapShot = await ref.child(userPath).once('value');
+			const userInfo = snapShot.exists() ? snapShot.val() : user;
+			setUser(userInfo);
```

Comment:
This was slightly confusing, it looks like you're storing user information or configs but you call the state variable user. and there's also a prop user.

Reply:
hmm, i think i wrote this along time ago. I dont think i need state here, since its just passing stuff along. Ill give this a rewrite

Reply:
also, is it possible to pass the snapshot as a prop? i make this call a second time in userList, seems like i could just hold onto snapshot and save a query.